health-monitors/gpu-health-monitor/gpu_health_monitor/dcgm_watcher/dcgm.py

```
@@ -459,46 +508,46 @@ def start(self, fields_to_monitor: list[str], exit: Event) -> None:
+        try:
+            while not exit.is_set():
+                # Wait for poll interval to allow DCGM initialization
+                log.debug("Waiting till next cycle")
+                exit.wait(self._poll_interval_seconds)
+
+                with metrics.overall_reconcile_loop_time.time():
+                    if dcgm_handle is None:
+                        try:
+                            dcgm_handle = self._get_dcgm_handle()
+                            dcgm_group, gpu_ids, _gpu_serials = self._initialize_dcgm_monitoring(dcgm_handle)
+                        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)